### PR TITLE
version to .2

### PR DESCRIPTION
**Pull request by harleenkohli**

### CODE REVIEW

File: src/build.props [hunk 1]
```diff
@@ -11,7 +11,7 @@
     <Copyright Condition=" '$(Copyright)' == '' ">© Microsoft Corporation. All rights reserved.</Copyright>
     <VersionPrefix>2.0.0</VersionPrefix>
     <VersionSuffix>csd.2.beta.2019-04-03</VersionSuffix>
-    <PackageVersionSuffix>.1</PackageVersionSuffix>
+    <PackageVersionSuffix>.2</PackageVersionSuffix>
```
Comment (lgolding) [inline src/build.props]: >.2 [](start = 26, length = 2)

Wow! Is that really all there is to it now? Nice!

Comment (harleenkohli) [reply src/build.props]: Actually! apologies! i am doing this for the first time, so! I looked at #1389 to see how Mike did it for previous version :/

---
In reply to: [272397727](https://github.com/Microsoft/sarif-sdk/pull/1404#discussion_r272397727) [](ancestors = 272397727)

Comment (lgolding) [reply src/build.props]: What about the version strings in the schema file?

---
In reply to: [272397727](https://github.com/Microsoft/sarif-sdk/pull/1404#discussion_r272397727) [](ancestors = 272397727)

Comment (lgolding) [reply src/build.props]: It looks like in #1389 Michael missed updating these lines:
```
  "title": "Static Analysis Results Format (SARIF) Version 2.0.0-csd.2.beta.2019-04-03 JSON Schema",
  "description": "Static Analysis Results Format (SARIF) Version 2.0.0-csd.2.beta-2019-04-03 JSON Schema: a standard format for the output of static analysis tools.",
```

---
In reply to: [272398201](https://github.com/Microsoft/sarif-sdk/pull/1404#discussion_r272398201) [](ancestors = 272398201,272397727)

Comment (lgolding) [reply src/build.props]: Also -- I think there is at least one package that has a custom .nuspec file where you have to change it.

---
In reply to: [272398650](https://github.com/Microsoft/sarif-sdk/pull/1404#discussion_r272398650) [](ancestors = 272398650,272398201,272397727)

Comment (lgolding) [reply src/build.props]: No, I'm wrong about that last one. All NuGet packages -- both the Multitool package, which has a custom .nuspec file, and all the other files, whose packages are generated directly from the csproj file -- all of them, I say, correctly pick up the version from build.props. So the only thing you need to fix is the schema.

---
In reply to: [272398781](https://github.com/Microsoft/sarif-sdk/pull/1404#discussion_r272398781) [](ancestors = 272398781,272398650,272398201,272397727)

Comment (lgolding) [reply src/build.props]: Hah! You not only have to fix the schema `title` and `description`, you have to fix the `version` property itself:
```
    "version": {
      "description": "The SARIF format version of this log file.",
      "enum": [ "2.0.0-csd.2.beta.2019-04-03" ]
    },
```

---
In reply to: [272401556](https://github.com/Microsoft/sarif-sdk/pull/1404#discussion_r272401556) [](ancestors = 272401556,272398781,272398650,272398201,272397727)

Comment (lgolding) [reply src/build.props]: There, that should be everything.

---
In reply to: [272401798](https://github.com/Microsoft/sarif-sdk/pull/1404#discussion_r272401798) [](ancestors = 272401798,272401556,272398781,272398650,272398201,272397727)

Comment (harleenkohli) [reply src/build.props]: Larry! we dont update schema file during minor updates, only major ones! according to Mike! @mikefan

---
In reply to: [272401866](https://github.com/Microsoft/sarif-sdk/pull/1404#discussion_r272401866) [](ancestors = 272401866,272401798,272401556,272398781,272398650,272398201,272397727)

**Pull request merged**